spec/app/lib/bing_api_v7_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+require 'bing_api_v7'
```

Comment:
no need to require file that doesn't exists
rails will auto-load your class when you `require 'rails_helper'`